src/Request.php

```diff
@@ -717,4 +719,51 @@ private static function limitTelegramRequests($action, array $data = [])
             }
         }
     }
+
+
+
+    private static function sendChatActionIfNecessary($action, $data)
+    {
+
+        if (!isset($data['sendChatAction'])) {
+            return false;
+        }
+
+        if ($data['sendChatAction'] === true) {
```

Comment:
Shouldn't this be `!== false`?

Reply:
Yeah that's my fault. it should be `!== true`